dictionaries/CallMap.php

```diff
@@ -13734,7 +13734,7 @@
 'str_replace' => ['string|string[]', 'search'=>'string|array', 'replace'=>'string|array', 'subject'=>'string|array', '&w_count='=>'int'],
 'str_rot13' => ['string', 'string'=>'string'],
 'str_shuffle' => ['string', 'string'=>'string'],
-'str_split' => ['list<string>|false', 'string'=>'string', 'length='=>'int'],
+'str_split' => ['non-empty-list<string>|false', 'string'=>'string', 'length='=>'int'],
```

Comment:
I think `$length` should be made a `positive-int` and `false` dropped, based on the docs

Reply:
Does positive-int exclude 0 too ?

Reply:
It does.

Reply:
I don't know if it's better to use a return type provider for the positive int check (cause the call is still possible).
But I've done the request change.

Reply:
In phpstan you would need `0|posivitve-int` to include 0

Reply:
> In phpstan you would need `0|posivitve-int` to include 0

So it's the same than psalm.

Here, 0 is not allowed for str_split, so it's perfect.